samples/XCT.Sample/Helpers/RelayCommand.cs

```diff
@@ -7,25 +7,21 @@ namespace Xamarin.CommunityToolkit.Sample
 {
 	public class RelayCommand : ICommand
 	{
-		readonly Action execute;
-		readonly Func<Task> asyncExecute;
+		readonly Action? execute;
```

Comment:
`execute` shouldn't be nullable (there are null checks for that in all constructors). Or maybe I missed something